lib/api/2.0/notification.js

```diff
@@ -30,7 +32,45 @@ var notificationProgressPost = controller(function(req, res){
     });
 });
 
+/**
+ * @api {post} /api/2.0/notification/alerts
+ * @apiDescription Receives Redfish alerts, refactors the message and publishes it into amqp
+ * @apiName notification post
+ * @apiGroup notification
+ */
+var twistedParser = parser.json({ type: 'application/x-www-form-urlencoded' });
+var jsonParser = parser.json();
+var notificationAlertsPost = controller({success: 201}, function(req,res) {
+    var format= req.headers["content-type"];
+    return Promise.try(function(){
+        return new Promise(function(resolve, reject) {
+            if (format === 'application/x-www-form-urlencoded'){
```

Comment:
From @brianparry 's explanation in email,  there is firmware bug that lead you have to do content-type parsing at here. Please add some comment here to let other developer be aware of such known issue.